lib/manager/actions/user.js

```diff
@@ -256,6 +257,12 @@ export function updateUserData (user: any, userData: any) {
           // If user being updated matches the logged in user, update their
           // profile in the application state.
           dispatch(userProfileUpdated(updatedProfile))
+        } else {
+          // a different user than the current user was updated. An action needs
+          // to be dispatched because the status message from the
+          // updatingUserData is still set to something, so we need to indicate
+          // that the request completed.
+          dispatch(clearStatusMessage())
```

Comment:
I wonder if rather than clearing the status message, we might want to replace the user profile that has been updated with the one contained in the store or alternatively, just fetch the user list again. Either way I think updating the user with its up-to-date info so that the user can verify the update matches their desired change is important.